### PR TITLE
Test Pull Request for review

### PR DESCRIPTION
**Pull request by segfaulthunter**

### CODE REVIEW

File: docs/analysis/index.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+### Events
+
+In the most general sense, a trace is simply a collection of "events" on a
+timeline. Events can have associated metadata and context which allows them to
+be interpreted and analysed.
+
+Events form the foundation of trace processor and are one of two types: slices
+and counters.
+
+<TODO add a picture from the UI>
```
Comment (primiano) [inline docs/analysis/index.md:16]: ...

File: docs/analysis/index.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+A track is a named partition of events of the same type and the same associated
+context. For example:
+
+- Scheduling slices have one track for each CPU
+- Sync userspace slice have one track for each thread which emitted an event
+- Async userspace slices have one track for each “cookie” linking a set of async
+  events
+
+The most intuitive way to think of a track is to imagine how they would be drawn
+in a UI; if all the events are in a single row, they belong to the same track.
```
Comment (primiano) [inline docs/analysis/index.md:46]: meh

Comment (segfaulthunter) [reply docs/analysis/index.md:46]: wow

Comment (chromy) [reply docs/analysis/index.md:46]: 👍 This is a great change 💯

File: docs/analysis/index.md (hunk 1 above)
Comment (chromy) [reply docs/analysis/index.md:16]: Hi!

Comment (chromy) [reply docs/analysis/index.md:16]: Hello?

**Pull request closed without merging**